oslo/torch/_C/csrc/CUDABinder.cpp

```diff
@@ -0,0 +1,455 @@
+#include <cuda_fp16.h>
```

Comment:
I think you should modify this file. 

1. because ngram_repeat_blcok kernel is nothing to do with LayerNorm.
2. this file name should be something like `LayerNormBinder.cpp`.

Reply:
OK, 
1. done removing <cuda_fp16.h> 
2. rename cpp file to `FusedLayerNormBinder.cpp`